__tests__/app.test.js

```diff
@@ -120,4 +120,48 @@ describe("GET /api/articles/:article_id", () => {
           });
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles", () => {
+    describe("Functionality", () => {
+      test("200: should return an object containing all articles including the following properties: author, title, article_id, topic, created_at, votes, art_img_url, comment_count, automatically sorted in reverse date order", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            body.articles.forEach((article) => {
```

Comment:
You need to be checking the length when we are expecting arrays to come back otherwise an empty array can cause a false passing test. Please implement this before merging.